projects/plugins/jetpack/modules/comments/base.php

```diff
@@ -249,15 +263,17 @@ function allow_logged_in_user_to_comment_as_guest( $comment_data ) {
 			'comment_author_email' => 'email',
 			'comment_author_url'   => 'url',
 		) as $comment_field => $post_field ) {
-			if ( $comment_data[ $comment_field ] != $_POST[ $post_field ] && 'url' != $post_field ) {
+			if ( $comment_data[ $comment_field ] !== $_POST[ $post_field ] && 'url' !== $post_field ) {
 				$author_change = true;
 			}
 			$comment_data[ $comment_field ] = $_POST[ $post_field ];
+			// phpcs:enable WordPress.Security.NonceVerification.Missing
 		}
```

Comment:
Even though it doesn't make any difference, I'd balance the comments (line 251 and here) with respect to the code blocks by doing like
```suggestion
		}
		// phpcs:enable WordPress.Security.NonceVerification.Missing
```

Reply:
Will add that, thanks!

Reply:
Will add it in manually (and line 251 as you mention), as GH 'dismissed' the review.